frontend/tests/playwright/fixtures/cart-drawer.ts

```diff
@@ -74,4 +74,12 @@ export class CartDrawer {
    async assertCartTotalQuantity(quantity: number): Promise<void> {
       expect(await this.getTotalQuantity()).toBe(quantity);
    }
+
+   async assertItemIsPresent(sku: string): Promise<void> {
```

Comment:
In our iFixit/ifixit code, present and visible are two different things. Visible means it has a height/width and is not `display:none` (I think). Present just means it's on the dom.

Should we change this function to be `assertItemIsVisible`? Or `assertItemInDrawer`?

I'm just now realizing this function reads as "assert item is present in cart drawer" not "assert item is present on dom." Hmm, I think this is fine.

Reply:
> Should we change this function to be assertItemIsVisible? Or assertItemInDrawer?

Since this is a method of `CartDrawer` then I was thinking that the context in which the item is present, can be understood as within the `CartDrawer`. 

However, it is true that we are only asserting that the item is in the DOM and displayed within the `cart-drawer` element. 

Should we change it to `assertItemIsVisible`?

Reply:
I think it's fine as is.